src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java

```diff
@@ -121,6 +127,11 @@ public String getColumnValue(final VariantContext variant) {
 
     AS_RAW_MQRankSum { // TODO -- maybe rely on 1/1 for call_GT, also get rid of the | at the beginning
```

Comment:
I haven't looked at the entirety of the code -- can this be subbed out with a constant also?